index.html

```diff
@@ -192,8 +192,47 @@ <h3 class="about-card-title">Skills</h3>
                 </ul>
             </div>
         </section>
-        
-        
+        <section class="contact-form">
+            <h2 class="contact-box contact-text">
+                I'm always interested in hearing about new projects,
+                 so if you'd like to chat please get in touch.
+            </h2>
+            <form action="https://formspree.io/f/mrgjojra" method="post" class="contact-form">
+                <div class="name">
+                    <input type="text" maxlength="30"  placeholder="Full name" required>
+                </div>
+                <div class="email">
+                    <input type="email" name="email"  placeholder="Email address" required>
+                </div>
+                <div >
+                    <textarea class="message" name="textarea"  maxlength="500" required placeholder=" Enter text here"></textarea>
+                </div>
+                <button type="submit" class="contact-btn contact-btn-text ">Get in touch</button>
```

Comment:
- [ ] As requested by the previous reviewer, kindly implement button states. These include on hover, on click, active etc. If you may not be sure of how to implement this, have a look at [this awesome resource](https://zellwk.com/blog/style-hover-focus-active-states/).
- [ ] It also seems like your button text is not fitting correctly in the button width. Kindly have a look at it.

**This is what I mean:**

![image](https://user-images.githubusercontent.com/86318284/186365087-9ca8b9b6-2e1c-4262-9090-9f52514aa220.png)